### PR TITLE
feat(subnavigation): add subnavigation

### PR DESCRIPTION
**Pull request by zwaardje**
Add subnavigation component

### CODE REVIEW

File: src/js/components/molecules/subnavigation-item/subnavigation-item.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<base-link :url="url" class="sub-nav__link">
+    <div class="sub-nav__image-container">
+        <c-image v-bind="image" keep-in-view></c-image>
+    </div>
+    <p class="sub-nav__label" :aria-label="label" > {{label}}</p>
```
Comment (rhelling) [inline src/js/components/molecules/subnavigation-item/subnavigation-item.html]: :space_invader:

File: src/js/utils/throw-props-plugin.js [hunk 2]
```diff
@@ -509,14 +509,15 @@ _gsScope._gsDefine("plugins.ThrowPropsPlugin", ["utils.VelocityTracker", "plugin
...
+
 	_gsScope._gsDefine("utils.VelocityTracker", ["TweenLite"], function() {
 
 		var _first,	_initted, _time1, _time2,
 			_capsExp = /([A-Z])/g,
 			_empty = {},
 			_doc = _gsScope.document,
 			_transforms = {x:1,y:1,z:2,scale:1,scaleX:1,scaleY:1,rotation:1,rotationZ:1,rotationX:2,rotationY:2,skewX:1,skewY:1,xPercent:1,yPercent:1},
-			_getComputedStyle = _doc.defaultView ? _doc.defaultView.getComputedStyle : function() {},
+			_getComputedStyle = _doc && _doc.defaultView ? _doc.defaultView.getComputedStyle : function() {},
```
Comment (rhelling) [inline src/js/utils/throw-props-plugin.js]: You changed vendor code ;-)

Comment (zwaardje) [reply src/js/utils/throw-props-plugin.js]: Yeah cause it bitched a lot

File: src/js/components/molecules/subnavigation-item/subnavigation-item.html (hunk 1 above)
Comment (zwaardje) [reply src/js/components/molecules/subnavigation-item/subnavigation-item.html]: 🏅

**Pull request merged**